docs/usage.rst

```diff
@@ -120,6 +120,13 @@ pseudo-elements::
     >>> selector.css('title::text').get()
     'Example website'
 
+You can extract inner text without specifying ``::text`` in your selector instead
+an optional paramter text=True in the ``get()`` or ``getall()`` methods. 
+
+    >>> selector.css('#images').get(text=True)
+
+You can pass additional paramter ``guess_punct_space``, ``guess_layout`` and ``guess_layout``
```

Comment:
```suggestion
To extract all text of one or more element and all their child elements, 
formatted as plain text taking into account HTML tags (e.g. ``<br/>`` is 
translated as a line break), set ``text=True`` in your call to 
:meth:`~Selector.get` or :meth:`~Selector.getall` instead of including 
``::text`` (CSS) or ``/text()`` (XPath) in your query: 

>>> selector.css('#images').get(text=True)
'Name: My image 1\nName: My image 2\nName: My image 3\nName: My image 4\nName: My image 5'

See :meth:`Selector.get` for additional parameters that you can use to change 
how the extracted plain text is formatted.
```